### PR TITLE
empty resource for testing

### PR DESCRIPTION
**Pull request by SergeyGaluzo**
!!!!! This PR is for testing only. Please don't review.

### CODE REVIEW

File: src/Microsoft.Health.Fhir.Core/Features/Persistence/ResourceRequest.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ public ResourceRequest(string method, Uri url = null)
         {
             EnsureArg.IsNotNullOrEmpty(method, nameof(method));
 
-            Url = url;
+            Url = url != null ? null : null;
```
Comment (PTaladay) [inline src/Microsoft.Health.Fhir.Core/Features/Persistence/ResourceRequest.cs:24]: Is this intended to always assign null? Or did you mean url != null ? url : null;

File: src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/FhirCosmosResourceWrapper.cs [hunk 2]
```diff
@@ -49,6 +49,7 @@ public FhirCosmosResourceWrapper(ResourceWrapper resource)
             IReadOnlyCollection<KeyValuePair<string, string>> lastModifiedClaims,
             string searchParameterHash = null)
             : base(resourceId, versionId, resourceTypeName, rawResource, request, lastModified, deleted, searchIndices, compartmentIndices, lastModifiedClaims, searchParameterHash)
+            ////: base(resourceId, versionId, ResourceKey.NameToId(resourceTypeName), rawResource, request, lastModified, deleted, searchIndices, compartmentIndices, lastModifiedClaims, searchParameterHash)
```
Comment (PTaladay) [inline src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/FhirCosmosResourceWrapper.cs:52]: Is it intentional to leave this here commented out? Same goes for the using statements and other lines in this file.

File: src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/Search/SearchIndexEntryJObjectGenerator.cs [hunk 3]
```diff
@@ -83,57 +91,83 @@ void ISearchValueVisitor.Visit(NumberSearchValue number)
...
+                AddProperty(SearchValueConstants.LowQuantityName, quantity.Low);
+                AddProperty(SearchValueConstants.HighQuantityName, quantity.High);
+            }
         }
 
         void ISearchValueVisitor.Visit(ReferenceSearchValue reference)
         {
             AddPropertyIfNotNull(SearchValueConstants.ReferenceBaseUriName, reference.BaseUri?.ToString());
-            AddPropertyIfNotNull(SearchValueConstants.ReferenceResourceTypeName, reference.ResourceType?.ToString());
+            ////AddPropertyIfNotNull(SearchValueConstants.ReferenceResourceTypeName, reference.ResourceType?.ToString());
```
Comment (PTaladay) [inline src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/Search/SearchIndexEntryJObjectGenerator.cs:121]: Same question here as I'm not sure of the context of this change. There are several items commented out. Shouldn't they just be removed?

File: src/Microsoft.Health.Fhir.Core/Features/Persistence/ResourceRequest.cs [hunk 4]
```diff
@@ -38,7 +38,7 @@ public ResourceRequest(Uri url, HttpMethod method)
         [JsonProperty("url")]
         public Uri Url { get; protected set; }
 
-        [JsonProperty("method")]
+        [JsonProperty("met")]
```
Comment (yoju0121) [inline src/Microsoft.Health.Fhir.Core/Features/Persistence/ResourceRequest.cs:41]: Same question here as above (changing from readable names to abbreviations)

**Comment by yoju0121** [comment]
Just a nit/perspective from someone that's completely new to the team - since there isn't a work item linked for this PR, a short description that provides some context of this change would be helpful in understanding :)

File: src/Microsoft.Health.Fhir.Core/Features/Persistence/KnownResourceWrapperProperties.cs [hunk 5]
```diff
@@ -7,42 +7,42 @@ namespace Microsoft.Health.Fhir.Core.Features.Persistence
 {
     public static class KnownResourceWrapperProperties
     {
-        public const string LastModified = "lastModified";
+        public const string LastModified = "lmd";
```
[the remark this answers is not in the document]
Comment (SergeyGaluzo) [reply src/Microsoft.Health.Fhir.Core/Features/Persistence/KnownResourceWrapperProperties.cs:10]: This is done to reduce storage footprint.

**Comment by SergeyGaluzo** [comment]
done

**Pull request closed without merging**